src/dlgTriggerEditor.cpp

```diff
@@ -3660,15 +3660,15 @@ void dlgTriggerEditor::addScript(bool isFolder)
     saveScript();
     QString name;
     if (isFolder) {
-        name = "New Script Group";
+        name = tr("New script group");
     } else {
-        name = "NewScript";
+        name = tr("New script");
     }
     QStringList mainFun;
     mainFun << "-------------------------------------------------\n"
             << "--         Put your Lua functions here.        --\n"
```

Comment:
This is a comment so it need not be in English - but we are not translating stuff in the Lua of items - so it needs either a `tr(`...`)` or a `QStringLiteral(`...`)` wrapper.

Reply:
Rebuilding this QStringList << barrage into a propper tr() was too big for me to include in scope here. I have now opened a new issue #2636 to tackle that.